modules/rostests/apitests/crt/msvcrt_crt_apitest.cmake

```diff
@@ -1371,6 +1371,13 @@ elseif(ARCH STREQUAL "amd64")
     #    sinf.c
     #    sqrtf.c
     )
+elseif(ARCH STREQUAL "arm")
+    list(APPEND SOURCE_MSVCRT
+        __rt_sdiv.c
+        __rt_udiv.c
+        __rt_sdiv64.c
+        __rt_udiv64.c
```

Comment:
Is this really worth 4 tests? It's all just software division.

Reply:
Ok I merged everything into a single test.